modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -8,10 +8,8 @@
 [id="change-default-storage-class_{context}"]
 = Changing the default storage class
 
-If you are using AWS, use the following process to change the default
-storage class. This process assumes you have two storage classes
-defined, `gp2` and `standard`, and you want to change the default
-storage class from `gp2` to `standard`.
+Use the following process to change the default storage class.
+For example you have two defined storage classes `gp2` and `standard`, and you want to change the default storage class from `gp2` to `standard`.
```

Comment:
Suggestion: add a comma after classes.

For example, you have two defined storage classes, `gp2` and `standard`, and...